src/utils/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const SMALL = 'small';
+export const MEDIUM = 'medium';
+export const LARGE = 'large';
+export const ORIGINAL = 'original';
+
+export const THUMBNAIL_SIZES = [SMALL, MEDIUM, LARGE, ORIGINAL];
```

Comment:
I don't think we need this here.

Reply:
This is needed because the thumbnails can't extend the download schema